Made doc modifications to the exercise steps (#6)
ansible-community/katacoda-scenarios: fixing-a-bug/step2.md

...
```

For more details on how to create your own Collection and also if you want to know about Collection Data Structure refer the link [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html).

## Setup the directory structure

Create the directory structure so `ansible` and `ansible-test` know which collection we are using.

`mkdir -p ansible_collections/community`{{execute}}

Go into the newly created directory:

`cd ansible_collections/community`{{execute}}

> For this training, we’ll use the existing `community.mysql` collection from Ansible GitHub.

Reviewer: ```suggestion
For this training, we’ll use the existing `community.mysql` collection available on GitHub.
```
